operations/security_and_authentication/Change_the_Keycloak_Admin_Password.md

```diff
@@ -1,46 +1,49 @@
-## Change the Keycloak Admin Password
+# Change the Keycloak Admin Password
 
-Update the default password for the admin Keycloak account using the Keycloak user interface \(UI\). After updating the password in Keycloak, encrypt it on the system and verify that the change was made successfully.
+Update the default password for the admin Keycloak account using the Keycloak user interface (UI). After updating the password in Keycloak, encrypt it on the system and verify that the change was made successfully.
 
-This procedure uses SYSTEM\_DOMAIN\_NAME as an example for the DNS name of the non-compute node \(NCN\). Replace this name with the actual NCN's DNS name while executing this procedure.
+This procedure uses SYSTEM\_DOMAIN\_NAME as an example for the DNS name of the non-compute node (NCN). Replace this name with the actual NCN's DNS name while executing this procedure.
 
-### Procedure
+## Procedure
 
-1.  Log in to Keycloak with the default admin credentials.
+1. Log in to Keycloak with the default admin credentials.
 
-    Point a browser at https://auth.cmn.SYSTEM_DOMAIN_NAME/keycloak/admin, replacing SYSTEM\_DOMAIN\_NAME with the actual NCN's DNS name.
+    Point a browser at `https://auth.SYSTEM_DOMAIN_NAME/keycloak/admin`, replacing SYSTEM\_DOMAIN\_NAME with the actual NCN's DNS name.
```

Comment:
I believe https://auth.cmn.SYSTEM_DOMAIN_NAME/keycloak/admin is correct for a 1.2 system.
For instance on mug this works
https://auth.cmn.mug.dv.cray.com/keycloak/admin
But this does not
https://auth.mug.dev.cray.com/keycloak/admin

Reply:
Thanks, I'll update it.